lib/tbot/output_utils.go

```diff
@@ -512,3 +514,21 @@ func makeNameOrDiscoveredNamePredicate(name string) string {
 		matchName, matchDiscoveredName,
 	)
 }
+
+// workloadAttrsForLog returns a string representation of the given WorkloadAttrs
+// for logging purposes.
+func workloadAttrsForLog(orig *workloadidentityv1.WorkloadAttrs) string {
```

Comment:
I really don't love this, and am open to suggestions!

Reply:
Should `WorkloadAttrs` implement [`LogValuer`](https://pkg.go.dev/log/slog#LogValuer)?

Reply:
That'd definitely be cleaner! Can we extend the generated protobuf structs without them getting [blown away](https://github.com/gravitational/teleport/blob/5447b96a794c467cc1a2438302d6001912646828/build.assets/genproto.sh#L37) by `make grpc`?

Reply:
Hm - I'm not sure we can easily extend the protobufs generated types unfortunately. We used to do this fairly often, but we tend to avoid that strategy now.

One thing that does come to mind is that if this func is used in a debug level message, we're going to be eagerly evaluating this rather than only evaluating this if the debug log level is in use. 

Perhaps it'd make more sense to have this function return a slog.Attr? I'm not sure how we can enforce that it is actually used though. We could introduce a Go type within `lib/tbot` that holds the underlying protobuf message and use that within the codebase instead. I'm sure there are other methods that would be helpful to hang from this.

Reply:
Great idea. I've added a wrapper type which implements `slog.LogValuer` in 3916d24b8925aa4d4776c9845c1cbd0e97bae2d8. See what you think 🙇🏻